src/components/ProjectMembersSummary/ProjectMemberSummary.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { Card } from '@material-ui/core';
+import React from 'react';
+import {
+  ProjectMembersSummary as ProjectMembersSummaryComponent,
+  ProjectMembersSummaryProps,
+} from './ProjectMembersSummary';
+import { ProjectMemberListFragment } from './ProjectMembersSummary.generated';
+
+export default { title: 'Components/ProjectMemberSummary' };
```

Comment:
The storybook title's never get modified, only the exports do. So we need to add the spaces ourselves. At least that's what we have been doing for the other components...
```suggestion
export default { title: 'Components/Project Member Summary' };
```

Reply:
I have template for storybook files and changed it so it will not have camel case. Shouldn't be in future PR's